src/components/pages/WelcomePage.tsx

```diff
@@ -780,31 +776,15 @@ class WelcomePage extends React.Component<Props, State> {
       return (
         <div
           onClick={ this.navigateTo(page.link || window.location.href) }
-          style={{ backgroundImage: `url(${ this.getAttachmentForPage(page) })` }}
+          style={{ backgroundImage: `url(${ page.featured_media })` }}
```

Comment:
are you sure page.featured_media is url and not id?

Reply:
These changes are from other branch. I've added changes to this branch and closed the old one.